nilearn/decoding/space_net.py

```diff
@@ -633,7 +633,7 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
          It is available only when parameter intercept is set to True.
 
     `cv_` : list of pairs of lists
-         Each pair are are the list of indices for the train and test
+         Each pair are the list of indices for the train and test
```

Comment:
I think this should be rephrased. Something like: `Each pair is composed of two lists of indices, one for the train, one for the test`.